cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -5,8 +5,11 @@ kind: ServiceAccount
 metadata:
   name: {{ template "kubecost.clusterControllerName" . }}
   labels:
-    {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
     app: {{ template "kubecost.clusterControllerName" . }}
+    app.kubernetes.io/name: {{ template "kubecost.clusterControllerName" . }}
+    helm.sh/chart: {{ include "cost-analyzer.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
For the sake of parity with the rest of this file, in this PR I suggest that we stick with just
```
{{ include "cost-analyzer.commonLabels" . | nindent 4 }}
```
(just deleting the specific `app` label, not replacing the common labels)

This is what the rest of this file uses (the Deployment [1], etc.) so we'll have a uniform experience, even if it isn't exactly correct.

I think it's worth changing these to be cluster controller-specific at some point, but we have quite a bit going on right now so I won't call it a priority.

[1] https://github.com/kubecost/cost-analyzer-helm-chart/blob/296ac4ef244168443e085c3e179511b8ed5409ea/cost-analyzer/templates/kubecost-cluster-controller-template.yaml#L170-L175